### DISCUSSION TITLE
Why is the background image missing or blank?

### DISCUSSION THREAD

**Discussion by AlfianMusthofa** [community/community#67388] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi guys, why doesn't the image on my GitHub page appear, aka it's blank?

Here's my pages [click please](https://alfianmusthofa.github.io/webSenal/WartegBahari/index.html)

Thank's guys.

**Comment by seyLu** [comment]
#### Hi @AlfianMusthofa,

![image](https://github.com/community/community/assets/98249191/c9497cf8-e17f-4150-a00c-5440879dc71f)


If you'd like to deploy a static html site to Github Pages, it is recommended to have an `index.html` at the root directory.

```bash
TicketID
  |-- ...
TravelCo
  |-- ...
WartegBahari
  |-- ...
index.html  # this will serve as entry file for your site
...
```

You can read more on [Github Pages: Creating your site](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site#creating-your-site)
